libs/dh/wholesale/feature-settlement-reports-v2/src/lib/table/dh-settlement-reports-table.component.html

```diff
@@ -55,7 +55,11 @@
     </ng-container>
 
     <ng-container *wattTableCell="columns['status']; header: t('columns.status'); let entry">
-      <dh-settlement-reports-status [status]="entry.statusType" />
+      <dh-settlement-reports-status
+        [status]="entry.statusType"
+        [reportId]="entry.id"
+        (download)="downloadReport($event)"
+      />
```

Comment:
suggestion: wouldn't it be better if the Download button was not part of `dh-settlement-reports-status`? It could simple be an `if/else` here depending on the status instead of going in a component only to send an event out.

Reply:
When we end up with logic around the status in two places instead of one. Right now the status component handle every case the status enum has.